assets/javascripts/kitten/karl/meters/random-progress.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Progress } from 'kitten/components/meters/progress'
+
+export class KarlRandomProgress extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      value: 0,
+    }
+  }
+
+  componentDidMount() {
+    const randomProgress = () => {
+      this.setState({ value: Math.random() * 100 })
+    }
+    const intervalId = setInterval(randomProgress, 1000)
+    this.setState({ intervalId })
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.state.intervalId)
```

Comment:
Du coup vis-à-vis de ce que j'ai dit avant, il faudrait `clearInterval(this.intervalId)`